plugins/modules/ipavault.py

```diff
@@ -261,7 +272,8 @@ def find_vault(module, name, username, service, shared):
 
 
 def gen_args(description, username, service, shared, vault_type, salt,
-             public_key, vault_data):
+             password, password_file, public_key, public_key_file, vault_data,
```

Comment:
I don't know if I'm missing something but it seems that password, password_file, vault_data and datafile_in are not used in the function. Do we actually need them here?

Reply:
Good catch. They were used during development, are not used anymore. Fixed.